### PR TITLE
Basic helm training

### PR DESCRIPTION
**Pull request by hairmare**
This is what I prepared a couple of moons ago... It probably makes sense to nitpick a bit ;)

### CODE REVIEW

File: kubernetes-workshop/6002_helm/modul_helm_6002_01_slides.md [hunk 1]
```diff
@@ -0,0 +1,464 @@
...
+* Ships with ready to use charts made by the community
+* best practices are baked into offical charts
+
+## What's wrong with kubectl?
+
+* Resources need to be mutated after a deploy
+* You can `kubectl edit <resource>`
+* But,  need to implement your own updating and rollback or depend on vendor specific tooling for lifecycle management
+
+**Helm helps with lifecycle managment while letting  define your infrastructure as code**
```
Comment (tongpu) [inline kubernetes-workshop/6002_helm/modul_helm_6002_01_slides.md]: better rephrase to "while allowing you to define your infrastructure as code"

File: kubernetes-workshop/6002_helm/modul_helm_6002_01_slides.md [hunk 2]
```diff
@@ -0,0 +1,464 @@
...
+Let's see how values get populated with a simple template.
+
+```bash
+echo  '{{ .Values.hello.world }}' > templates/hello.tpl
+```
+
+## Default values are in a the charts `values.yaml`
+
+```yaml
+# values.yml
```
Comment (tongpu) [inline kubernetes-workshop/6002_helm/modul_helm_6002_01_slides.md]: s/yml/yaml/

File: kubernetes-workshop/6002_helm/modul_helm_6002_01_slides.md [hunk 3]
```diff
@@ -0,0 +1,464 @@
...
+  {{- end -}}
+# ...
+```
+
+## Debugging Templates
+
+```bash
+# verify that chart follows best practices
+helm lint
+# let server sender tempaltes and return resulting manifest
```
Comment (tongpu) [inline kubernetes-workshop/6002_helm/modul_helm_6002_01_slides.md]: s/tempaltes/templates/

**Comment by hairmare** [comment]
The requested changes are addressed in https://github.com/adfinis-sygroup/adsy-trainings/pull/14/commits/7f20338a792ac36669a3291006e907b53d81910a.

**Comment by tongpu** [comment]
LGTM

**Pull request merged**